Software/scan_key/AuthenticationSQL.py

```diff
@@ -62,8 +61,34 @@ def twoFactorAuth(id, pin, cursor):
         return(False) #if pin is invalid, then not allowed
 
 def getID(idString, cursor):
-    cursor.execute("SELECT CUID, pin FROM USER WHERE t1String = " + idString) #getting user w/ the id
+    cursor.execute("SELECT CUID FROM USER WHERE t1String = " + idString) #getting user w/ the id
     data = cursor.fetchone() #fetching data into array
```

Comment:
There's something with the encoding of the string that won't allow it to be used in the SQL instruction so I had to use '%s'. Or I could do
" ' " + currMachineID + " ' "